service/raft/raft_group_registry.cc

```diff
@@ -92,7 +92,7 @@ void raft_group_registry::init_rpc_verbs() {
             auto& rpc = self.get_rpc(gid);
             // The address learnt from a probably unknown server should
             // eventually expire
-            rslog.debug("Adding address {} -> {} to the address map", from, addr);
+            rslog.debug("raft_group_registry - Adding address {} -> {} to the address map", from, addr);
```

Comment:
Doesn't every logger already prepend every log message with `<logger-name> -`?

Reply:
but not class name.

Reply:
But here we will get a message
```
raft_group_registry - raft_group_registry - Adding address ...
```

because the logger is named `raft_group_registry`.
Why the double `raft_group_registry -`?

Reply:
Removed.